WalletWasabi.Gui/MainWindow.xaml

```diff
@@ -21,7 +21,28 @@
                       UseLayoutRounding="True" RenderOptions.BitmapInterpolationMode="HighQuality"
                       id:DragBehavior.IsEnabled="False" id:DropBehavior.IsEnabled="False">
   <cont:MetroWindow.TitleBarContent>
-    <menu:MainMenuView IsVisible="{Binding !$parent[cont|MetroWindow].DataContext.LockScreen.IsLocked}" DataContext="{Binding Shell.MainMenu}" Margin="4 0 0 0"  Foreground="{DynamicResource ThemeForegroundBrush}" VerticalAlignment="Stretch" FontSize="13" />
+    <StackPanel Orientation ="Horizontal" IsVisible="{Binding !$parent[cont|MetroWindow].DataContext.LockScreen.IsLocked}">
+      <menu:MainMenuView DataContext="{Binding Shell.MainMenu}" Margin="4 0 0 0"  Foreground="{DynamicResource ThemeForegroundBrush}" VerticalAlignment="Stretch" FontSize="13" />
+      <Button Command="{Binding LockScreenCommand}" Margin="10 0">
+        <StackPanel Orientation="Horizontal" Spacing="8">
+          <Grid Height="10" Width="10">
+            <DrawingPresenter VerticalAlignment="Center" HorizontalAlignment="Center">
+              <DrawingPresenter.Drawing>
+                <DrawingGroup>
+                  <DrawingGroup.Children>
```

Comment:
@jmacato do you know how to move this icon back to Icons.xaml in an elegant way to make this xaml easier to read? Something like a StaticResource or...?

Reply:
@molnard hmm i'll try moving it to Icons.xaml; StaticResource should work okay :+1: But iirc that icon was in Icons.xaml before... not sure how it got there

Reply:
We decided to merge it now as is. But this could be a useful goal for another PR.

Reply:
Thanks! Though i should make that PR now... Having that icon there bothers me a bit too :)